editor/plugins/tiles/tiles_editor_plugin.cpp

```diff
@@ -127,6 +130,8 @@ void TilesEditorUtils::_thread() {
 				// Add the viewport at the last moment to avoid rendering too early.
 				callable_mp((Node *)EditorNode::get_singleton(), &Node::add_child).call_deferred(viewport, false, Node::INTERNAL_MODE_DISABLED);
 
+				MessageQueue::get_singleton()->flush();
```

Comment:
Unsure if the better style is this or:
```suggestion
				queue.flush();
```

Reply:
An interesting question. I'd favor the cumbersome syntax, because, even if in this case you have the pseudo-singleton variable at hand, a more explicit syntax is more consistent with the typical usage of message queues.